contracts/chain-adapters/interfaces/ArbitrumForwarderInterface.sol

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+/**
+ * @title Staging ground for incoming and outgoing messages
+ * @notice Unlike the standard Eth bridge, native token bridge escrows the custom ERC20 token which is
+ * used as native currency on L3.
+ * @dev Fees are paid in this token. There are certain restrictions on the native token:
+ *       - The token can't be rebasing or have a transfer fee
+ *       - The token must only be transferrable via a call to the token address itself
+ *       - The token must only be able to set allowance via a call to the token address itself
+ *       - The token must not have a callback on transfer, and more generally a user must not be able to make a transfer to themselves revert
+ *       - The token must have a max of 2^256 - 1 wei total supply unscaled
+ *       - The token must have a max of 2^256 - 1 wei total supply when scaled to 18 decimals
+ */
+interface ArbitrumERC20Bridge {
```

Comment:
nit: we probably want to move these arb interfaces to contracts/external/interfaces, since they are used in the Arbitrum adapter as well. Keeps files succinct, reduces chances of typos, etc

Reply:
Done here. https://github.com/across-protocol/contracts/pull/609/commits/0de93033f590bd7886fae198bf05dc4b36e9732e